sdk/communication/azure-communication-administration/src/main/java/com/azure/communication/administration/PhoneNumberAsyncClient.java

```diff
@@ -896,4 +896,67 @@
         };
 
     }
+
+ /**
+     * Releases the given phone numbers.
+     * This function returns a Long Running Operation poller that allows you to 
+     * wait indefinitely until the operation is complete.
+     * 
+     * @param phoneNumbers A list of {@link PhoneNumber} with the desired numbers to release
+     * @param pollInterval The time our long running operation will keep on polling 
+     * until it gets a result from the server
+     * @return A {@link PollerFlux} object with the release entity
+     */
+    @ServiceMethod(returns = ReturnType.COLLECTION)
+    public PollerFlux<PhoneNumberRelease, PhoneNumberRelease> 
```

Comment:
I am not sure what the developers will understand out of PhoneNumberRelease, can't we have a better naming atleast for this?

Reply:
Ideally, I would want to see something more meaningful -- do we know how other languages are solving this?